tests/test_semantic_segmentation.py

```diff
@@ -0,0 +1,23 @@
+"""Unit test related to semantic segmentation model instanciation
+"""
+
+from keras.models import Model
+
+from deeposlandia.keras_semantic_segmentation import SemanticSegmentationNetwork
+
+def test_network_instanciation():
+    """Test the instanciation of a simple feature detection network
```

Comment:
Could you remove the word "instanciation"?

Reply:
Done.